### ISSUE TITLE
Use case: Existing content

### ISSUE THREAD

**Issue by jakearchibald** [bkardell/outline#1]
I have an existing CMS that is full of flat-outline content. I want to transition to using nested outlines from now on, but I don't want to have to support two models.

Ideally, with a template like this:

```html
<h1>Important news!</h1>
{articles.forEach(article => 
  <article>{article}</article>
)}
```

…articles could be a mix of nested outlines and flat outlines without breaking usability.

**Comment by bkardell** [comment]
Yeah I agree existing, flat content is a use case that all proposals need to answer what is affected/how when fed this.  I already created https://github.com/bkardell/outline/blob/master/use-cases/0.html which is just totally flat, but would you like to create a 1.html that shows a mix of some kind (or even several if you think that is apt) during what such a transition might look like?

It's pretty much doable through the gh web ui, but if you would rather just paste it here I can add it

**Comment by bkardell** [comment]
merged... do you think we close these and just say the items in the use case folder are the inputs you have to answer for or do we leave these open?

**Comment by jakearchibald** [comment]
My only concern is "success" might not be well captured by the use-cases.

**Comment by bkardell** [comment]
well, I'm all ears for thoughts and ideas.  I'll leave this open for now.

**Comment by bkardell** [comment]
@jakearchibald I am not sure if you think that this belongs under 'existing content' or deserves it's own use case ('existing content' is a big umbrella as phrased) but I added a 3.html which contains another thing I've seen a lot, mismatch between what the DOM is saying and what the heading levels appear to be saying

**Comment by bkardell** [comment]
I feel like we also need some use cases which display:none some headings such that we have 

h1
...
h2
...
h3
...
h2

and then the first h2 is hidden, etc... because if we are re-numbering, I'd like to know how that exposes to the AT tree.
